ibm/service/kubernetes/resource_ibm_container_cluster.go

```diff
@@ -421,29 +385,6 @@ func ResourceIBMContainerCluster() *schema.Resource {
 				DiffSuppressFunc: flex.ApplyOnce,
 			},
 
-			"org_guid": {
-				Description: "The bluemix organization guid this cluster belongs to",
-				Type:        schema.TypeString,
-				Optional:    true,
-				Deprecated:  "This field is deprecated",
-			},
-			"space_guid": {
-				Description: "The bluemix space guid this cluster belongs to",
-				Type:        schema.TypeString,
-				Optional:    true,
-				Deprecated:  "This field is deprecated",
-			},
-			"account_guid": {
-				Description: "The bluemix account guid this cluster belongs to",
-				Type:        schema.TypeString,
-				Optional:    true,
-				Deprecated:  "This field is deprecated",
-			},
-			"wait_time_minutes": {
-				Type:       schema.TypeInt,
-				Optional:   true,
-				Deprecated: "This field is deprecated",
-			},
```

Comment:
terraform plan will throw out  Error: Unsupported argument
however I was thinking it is ok to remove as even if you have them there, they do nothing, and might suggest otherwise at first glance. 
If it would still work in any capacity, despite being deprecated, i would keep it. 
But this way I think it is also misleading, to be able to have it in a tf, and expect it to work as it used to. but does nothing. 

But ofc, it can be restored.

Reply:
I'm in favour of removing them, but just wanted to understand what our uses will see.